src/components/skills-quiz/JobDescriptions.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {
+  useEffect, useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { InfoOutline } from '@edx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+import { Skeleton } from '@edx/paragon';
+import { fetchJobPathDescription } from './data/service';
+import { DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE, JOB_DESCRIPTION_DISCLAIMER } from './constants';
+
+const JobDescriptions = ({
+  currentJobID, futureJobID, currentJobDescription, futureJobDescription, goal,
+}) => {
+  const [isLoadingJobPathDescription, setIsLoadingJobPathDescription] = useState(false);
```

Comment:
suggestion: `getJobPathDescription` is a good candidate for a custom hook.

Reply:
@saleem-latif added a custom hook, please have a look.